### ISSUE TITLE
Training & Test

### ISSUE THREAD

**Issue by strutive07** [strutive07/transformer-tensorflow2.0#7]
## 1. without label smoothing.
WMT14 en_de data

GPU: P100 * 4
TRAIN_RATIO = 0.9
D_POINT_WISE_FF = 2048
D_MODEL = 512
ENCODER_COUNT = DECODER_COUNT = 6
EPOCHS = 20
ATTENTION_HEAD_COUNT = 8
DROPOUT_PROB = 0.1
BATCH_SIZE = 32 * 4
SEQ_MAX_LEN_SOURCE = 100
SEQ_MAX_LEN_TARGET = 100
BPE_VOCAB_SIZE = 32000

BLEU Score: 24.28
checkpoint: https://drive.google.com/file/d/1hy1FmXqKlaXhEGO9gpAU3K9zkv1EV6aa/view?usp=sharing
